src/workers/notifications/index.ts

```diff
@@ -62,10 +60,9 @@ export function notificationWorkerToWorker(
         });
       } catch (originalError) {
         const err = originalError as TypeORMQueryFailedError;
-
         if (err?.code === TypeOrmError.NULL_VIOLATION) {
           logger.error(
-            { data: messageToJson(message), err },
+            { data: parsedMessage, err },
```

Comment:
This actually would break now, was surprised we didn't have any issues yet 😅

Reply:
<img width="474" height="474" alt="image" src="https://github.com/user-attachments/assets/727a1a63-fb4e-4732-98b9-af422807c1b3" />